source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -93,6 +93,68 @@ The operations are categorized by purpose and functionality.
    from the Query API expression provided in each example. However,
    both expressions will produce the same aggregation result.
 
+Arithmetic Operations
+~~~~~~~~~~~~~~~~~~~~~
+
+You can use the methods described in this section to perform arithmetic
+operations on values of type ``MqlInteger`` or ``MqlNumber``.
+
+The following table lists the arithemetic methods
+available in the driver and corresponding expressions in the
+Query API. The methods link to API documentation and the Query API
+expressions link to Server manual documentation.
```

Comment:
Suggestion: describe what users will find in the Server docs

```suggestion
The following table describes arithmetic operation methods available in the driver and corresponding expression operators in the Query API. The method names link to API documentation and the aggregation pipeline operator names link to descriptions and examples in the Server manual documentation.
```

Reply:
Per our discussion, this should move into the string pull request.